eras/alonzo/formal-spec/utxo.tex

```diff
@@ -18,7 +18,7 @@ \section{UTxO}
     & \fun{feesOK} : \PParams \to \Tx \to \UTxO \to \Bool  \\
     & \fun{feesOK}~\var{pp}~tx~utxo~= \\
     &~~      \minfee{pp}~{tx} \leq \txfee{txb} \wedge (\fun{txrdmrs}~tx \neq \emptyset \Rightarrow \\
-    &~~~~~~((\forall (a, \wcard, \wcard) \in \fun{range}~(\fun{collateral}~{txb} \restrictdom \var{utxo}), a \in \AddrVKey) \\
+    &~~~~~~(\forall (a, \wcard, \wcard) \in \fun{range}~(\fun{collateral}~{txb} \restrictdom \var{utxo}), a \in \AddrVKey) \\
```

Comment:
```suggestion
    &~~~~~~((\forall (a, \wcard, \wcard) \in \fun{range}~(\fun{collateral}~{txb} \restrictdom \var{utxo}), a \in \AddrVKey) \\
```

Reply:
I think it might be clearer to instead keep those parens, but close the outermost one. yea?

Reply:
I think the line breaks already help with noticing the conjunctions, so I'd prefer if we removed the parentheses

Reply:
hmm. but the line break isn't really semantically meaningful, I feel like the parens are much more explicit. let's see what @WhatisRT thinks.

Reply:
conjunction binds more strongly than implication so the parentheses wouldn't change anything

Reply:
I'd happily remove even the pair of parentheses around the forall line as well ;)

Reply:
ah, I was worried about the scope of the quantifier, but that's still covered with this deletion. great!